src/components/calcite-input-date-picker/usage/Basic.md

```diff
@@ -0,0 +1,26 @@
+```html
+<div style="width: 400px">
+  <calcite-label
+    layout="inline"
+    class="sc-calcite-label-h sc-calcite-label-s"
```

Comment:
@benelan can we remove the class here? I think these are added by stencil because they are scoped components currently. Applies to other usage snippets.

Reply:
Nice catch, I removed those classes from most of the snippets but missed that one. I searched all of the usage examples, the rest are fine